package.json

```diff
@@ -18,7 +18,7 @@
     "format:fix": "turbo format --continue -- --write --cache --cache-location node_modules/.cache/.prettiercache  --ignore-path ./.prettierignore",
     "lint": "turbo lint --continue -- --cache --cache-location node_modules/.cache/.eslintcache",
     "lint:fix": "turbo lint --continue -- --fix --cache --cache-location node_modules/.cache/.eslintcache",
-    "lint:ws": "npx sherif@0.10.0",
+    "lint:ws": "npx sherif@0.10.0 -i next",
```

Comment:
can i clarify what -i next does?

Reply:
it tells `sherif` to ignore the different `next` versions and not raise it as error

this is needed because `postinstall` runs `lint:ws` and block `npm i` from proceeding